ds3_java_cli/src/main/java/com/spectralogic/ds3cli/command/EjectStorageDomain.java

```diff
@@ -0,0 +1,85 @@
+/*
+ * *****************************************************************************
+ *   Copyright 2014-2016 Spectra Logic Corporation. All Rights Reserved.
+ *   Licensed under the Apache License, Version 2.0 (the "License"). You may not use
+ *   this file except in compliance with the License. A copy of the License is located at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   or in the "license" file accompanying this file.
+ *   This file is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR
+ *   CONDITIONS OF ANY KIND, either express or implied. See the License for the
+ *   specific language governing permissions and limitations under the License.
+ * ***************************************************************************
+ */
+
+package com.spectralogic.ds3cli.command;
+
+import com.google.common.collect.ImmutableList;
+import com.spectralogic.ds3cli.Arguments;
+import com.spectralogic.ds3cli.exceptions.CommandException;
+import com.spectralogic.ds3cli.models.DefaultResult;
+import com.spectralogic.ds3client.commands.spectrads3.DeletePermanentlyLostTapeSpectraS3Request;
+import com.spectralogic.ds3client.commands.spectrads3.EjectStorageDomainSpectraS3Request;
+import com.spectralogic.ds3client.utils.Guard;
+import org.apache.commons.cli.MissingOptionException;
+import org.apache.commons.cli.Option;
+
+import java.io.IOException;
+import java.util.UUID;
+
+public class EjectStorageDomain extends CliCommand<DefaultResult> {
+
+    private UUID id;
+    private String ejectLabel;
+    private String ejectLocation;
+    private String bucket;
+
+    public EjectStorageDomain() {
+    }
+
+    @Override
+    public CliCommand init(final Arguments args) throws Exception {
+        this.id = UUID.fromString(args.getId());
```

Comment:
You're doing the null check after trying to convert the value.  This may throw a NPE